Glimpse.ApplicationInsights/GlimpseTelemetryChannel.cs

```diff
@@ -15,7 +16,13 @@ public class GlimpseTelemetryChannel : ITelemetryChannel
         [ThreadStatic]
         private static Stopwatch fromLastWatch;
 
+        /// <summary>
+        /// Initializes the channel from ApplicationInsights.config file.
+        /// </summary>
+        public ITelemetryChannel Channel { get; private set; }
```

Comment:
Can you please name it ApplicationInsightsChannel or ForwardChannel so it will be clearer in configuration what is being configured and why one channel has another channel as a property